### DISCUSSION TITLE
1.28.3-1.28.4

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#15278] [General]
# Notes
- For CC2652/CC1352 ti.router users: a [new firmware](https://github.com/Koenkk/Z-Stack-firmware/tree/develop/router/Z-Stack_3.x.0/bin) is available which allows setting the transmit power.
- For supervised Home Assistant users running Zigbee2MQTT **not** via the addon: a [proxy addon](https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/tree/master/zigbee2mqtt-proxy) is available now

# Improvements
- [#604](https://github.com/Koenkk/zigbee-herdsman/pull/604) #14600 #15006 EZSP adapter improvements
- #14905 Support Home Assistant MQTT update
- #14915 Update Docker image to use `node:18-alpine3.16`
- [#341](https://github.com/Koenkk/Z-Stack-firmware/issues/341) Support `transmit_power` for ti.router
- #14287 Expose `power_on_behavior` for lights supporting it
- [#4876](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4876) Support `power_outage_memory` for TuYa TS0001_power
- #14682 Support `battery` for OSRAM AC01353010G
- [#175](https://github.com/Koenkk/zigbee-OTA/pull/175) Support OTA for SONOFF S40ZBTPB
- [#4881](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4881) Support cooling mode for SMaBiT (Bitron Video) AV2010/32
- #14788 Remove core-js polyfills (drops Node.js 10 support)
- #14427 Support dry `system_mode` for Namron 4512737/4512738
- [#4888](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4888) Support OTA for Develco AQSZB-110
- #14833 Support Node.js 19
- [#4777](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4777) Add `action_brightness_delta` for simulated brightness feature
- [#4910](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4910) Support more features for Inovelli VZM31SN
- #11374 Support OTA for Innr SP 234
- #14865 Add more icons for Home Assistant discovery
- #14179 Update `local_temperature` when `local_temperature_calibration` is changed for Moes BRT-100-TRV
- #14283 Log `Zigbee2MQTT started!` on startup
- [#4929](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4929) Improve GiEX QT06 exposes
- [#4932](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4932) Support `switch_type` for TuYa TS0001_power
- #14953 Allow to specify data paths out of Zigbee2MQTT directory
- #14955 Allow to use `!secret.yaml` instead of just `!secret` in `configuration.yaml`
- [#180](https://github.com/Koenkk/zigbee-OTA/pull/180) [#189](https://github.com/Koenkk/zigbee-OTA/pull/189) [#191](https://github.com/Koenkk/zigbee-OTA/pull/191)  [#192](https://github.com/Koenkk/zigbee-OTA/pull/192) [#193](https://github.com/Koenkk/zigbee-OTA/pull/193) [#197](https://github.com/Koenkk/zigbee-OTA/pull/197) [#198](https://github.com/Koenkk/zigbee-OTA/pull/198) Support OTA for Third Reality 3RTHS24BZ, 3RDS17BZ, 3RMS16BZ, 3RSB22BZ, 3RSS009Z , 3RSP019BZ  and 3RSB015BZ
- #14812 Support `auto_mode` for Legrand 412171
- #15014 Support `energy` and `power` for GE 45856GE
- #14369 Don't log undefined datapoints as error for TuYa
- Support OTA for Innr SP 224
- [#4979](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4979) Support `unoccupied_heating_setpoint` for Acova PERCALE2, ALCANTARA2 and TAFFETAS2
- [#4818](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4818) Support more actions (`action`) for Namron 4512706
- [#4986](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4986) Add `unoccupied_heating_setpoint` to Sinope thermostats
- #15126 Add polling when bounded for Trust and Gledopto
- [#4999](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4999) Allow reading of `card`, `energy`, `power` for Dawon DNS
- [#186](https://github.com/Koenkk/zigbee-OTA/pull/186) Support OTA for Third Reality 3RWS18BZ
- [#5006](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5006) Support electrical measurements for Sinopé SP2600ZB
- [#5011](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5011) Support `system_mode` `away` for TuYa TS0601_thermostat
- [#188](https://github.com/Koenkk/zigbee-OTA/pull/188) Support OTA for Third Reality 3RTHS24BZ
- [#195](https://github.com/Koenkk/zigbee-OTA/pull/195/) Support OTA for Xiaomi SRTS-A01
- [#5032](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5032) Support `power_on_behavior` for all TuYa switches
- #15237 Disable `update_available` sensor in Home Assistant by default
- #15239 Support both gangs for Schneider Electric WDE002906
- [#5050](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5050) Support gradient scenes for Philips 929003535301
- [#5028](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5028) Support more feature for Bosch BTH-RA
- [#5049](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5049) Support `energy` for Ubisys D1

# Fixes
- #14396  Fix starting via `cli.js`
- [#4869](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4869) Increase max `current_heating_setpoint` for Moes BHT-002-GCLZB to 35
- [#4873](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4873) Align `battery_low` with ZCL spec 
- #14750  Fix no individual control of gangs of TuYa TS0012_switch_module
- [#4877](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4877) Revert add `emergency_heating` for Hive SLR2b/c
- [#4879](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4879) Fix Namron 540139X hysterersis value range
- #14785 Fix `ppm` and `formaldehyd` units of TuYa TS0601_smart_air_house_keeper 
- [#4887](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4887) Fix Moes BHT-002-GCLZB `max_temperature_limit` range
- #14809 Fix `temperature` and `battery` reporting for OWON THS317-ET
- #14803 Fix cover state for BTicino K4027C/L4027C/N4027C/NT4027C
- #14779 Fix IKEA TRADFRI bulbs losing their `level_config.on_level` value
- #14804 Fix TuYa TV02-Zigbee schedule temperature value
- #14356 Fix no `energy` value for Lidl HG08673-FR
- #14771 Fix no `system_mode` for Moes BRT-100-TRV
- [#4909](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4904) Fix `led_color_on` description
- [#4905](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4905) Fix power source for Hampton Bay 99432
- [#4908](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4908) Identify `LXN56-DS27LX1.3` as LXZB-02A instead of HGZB-04D / HGZB-4D-UK
- #13929 Fix `energy` and support `power_on_behavior` and `switch_type` for TuYa TS000F_power
- #13800 Fix `dpEntry[2].from is not a function` error for TuYa TS0601_dimmer_1
- #14857 #15026 Fix configure failing for some TuYa devices (`UNSUPPORTED_ATTRIBUTE ` error)
- #14877 Fix power source for LEDVANCE AC26940/AC31266
- [#4834](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4834) Fix Leviton ZS057 state reporting
- [#4916](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4916) Fix Yookee D10110 cover state
- [#4918](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4918) Fix hystersis setting for Namron 540139X
- [#4922](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4922) Align `battery_low` computation with ZCL spec
- #14920 Fix power source of Bouffalolab RMC002
- [#4942](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4942) Fix no `energy` reported for Innr SP 234 
- [#4945](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4945) Fix Ubisys `input_configurations` reading
- [#4882](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4882) Fix Owon PIR313-E `temperature`, `humidity` and `illuminance` reporting
- #14932 Fix Ubisys `configure_device_setup` cannot be read
- #14817  Disable unsupported move/step and transition for Lonsonho QS-Zigbee-D02-TRIAC-LN and TS110E_2gang
- #13570 Fix brightness changing to 1 when light is turned on for some lights
- [#4950](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4950) Fix Sinope TH1123ZB and TH1124ZB value publishing
- #14934 Fix no energy measurements for TuYa TS011F_plug_3 with application version 160
- [#4964](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4964) Fix setting `unoccupied_cooling_setpoint` resulting in `unoccupied_heating_setpoint` update in state
- #13329 Fix incorrect `device_temperature`, `energy`, `power` and `temperature` values for Develco devices
- [#15025](https://github.com/Koenkk/zigbee2mqtt/issues/15025) Fix crash and configure failing for Sunricher SR-ZG9092A
- #15101 Fix incorrect cover state for Legrand 067776
- #15173  Fix invalid temperature values for OWON THS317-ET
- #14853 Fix memory issue with TuYa electrical measurement polling
- [#1723](https://github.com/Koenkk/zigbee2mqtt.io/pull/1723) Fix max `current_summation` value for Develco ZHEMI101
- [#5014](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5014) Fix battery reporting for TuYa TS0215A_sos
- [#5029](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5029) Fix `min_temperature_limit` not updating for Moes BHT-002
- #14190 Fix TuYa TS0202_2 `light`, `action` and `occupancy`
- [#5053](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5053) Fix Third Reality 3RWS18BZ battery % divided by 2
- [#1734](https://github.com/Koenkk/zigbee2mqtt.io/pull/1734) Remove unsupported pm25 options from TuYa TS0601_air_quality_sensor

# New supported devices
This release adds support for **54** devices:
- `046677577957` Philips Hue White and Color Ambiance BR30 with bluetooth
- `046677578138` Philips Hue White ambiance BR30 E26
- `12242` EGLO ST64 adjustable white filament bulb
- `4058075208384` LEDVANCE SMART+ Classic A60 E27 Tunable white
- `4058075729001` LEDVANCE SMART+ CL A60 E27 Tunable white
- `4058075729124` LEDVANCE SMART+ CL P40 E14 Tunable white
- `4058075729186` LEDVANCE SMART+ Spot PAR16 28 GU10 Multicolor
- `4058075729261` LEDVANCE SMART+ Plug EU
- `4058075729285` LEDVANCE SMART+ Plug UK
- `412173` Legrand DIN dry contactor module
- `45727` Müller Licht Tint Amela 42cm, white+ambiance (1800-6500K)
- `45728` Müller Licht Tint bulb E27 white+color
- `45730` Müller Licht Tint candle E14 white+color
- `500.44` Paulmann URail power supply
- `5060831P7` Philips Hue White & Color ambience Centris ceiling light (3 spots)
- `579573` Philips Hue White and Color Ambiance Slim Downlight 6"
- `900024` EGLO SALITERAS-Z ceiling light
- `900091` EGLO ROVITO-Z ceiling light
- `915005987101` Philips Hue Gradient Signe floor lamp (white)
- `915005987601` Philips Hue Gradient Signe floor lamp (black)
- `929002422901` Philips Hue Play gradient lightstrip 75
- `9290024691A` Philips Hue white single filament bulb A19 E26 with Bluetooth (1100 Lumen)
- `9290031452` Philips Hue white ambiance filament E14 (with Bluetooth)
- `929003535301` Philips Hue Festavia gradient light string 250
- `AC33901` LEDVANCE SMART+ Classic B40 E14 Tunable white
- `BN-600078` Dresden Elektronik Zigbee controller for 1-10V/PWM
- `BTH-RA` Bosch Radiator thermostat II
- `E2D-G73` Sengled Smart window and door sensor G2
- `GL-G-101P` Gledopto Zigbee 12W garden lamp RGB+CCT (pro)
- `HSE2927E` Datek Eva Zigbee motion sensor
- `HT402` Stelpro Hilo thermostat
- `LDHD2AZW` Leedarson Magnetic door & window contact sensor
- `LED2006R9` IKEA TRADFRI E26 PAR38 LED bulb 900 lumen, dimmable, white spectrum, downlight
- `M423-9E` Akuvox Smart temperature & humidity Sensor
- `MEG5161-0000` Schneider Electric Merten PlusLink relay insert with Merten Wiser system M push button (1fold)
- `ML-ST-BP-DIM` M-ELEC Stitchy dim mechanism
- `OFL 142 C` Innr Outdoor flex light colour LED strip 4m, 1440lm, RGBW
- `PIR323-PTH` OWON Multi-sensor
- `RB 286 C` Innr E27 bulb RGBW
- `SM308-S` Samotech Zigbee in wall smart switch
- `SM309-S` Samotech Zigbee dimmer 400W
- `TS000F_power` TuYa Switch with power monitoring
- `TS0601_3_phase_clamp_meter` TuYa 3-phase clamp power meter
- `TS0601_din_2` TuYa Zigbee DIN energy meter
- `TS0601_smart_human_presence_sensor` TuYa Smart Human presence sensor
- `TS0601_thermostat_2` TuYa Thermostat radiator valve
- `TS110E` TuYa 1 channel dimmer
- `TWV` UHome Smart valve
- `WSP402` OWON Smart plug
- `ZBMINIL2` SONOFF Zigbee smart switch (no neutral)
- `ZG-102ZL` TuYa Luminance door sensor
- `ZG-225Z` TuYa Gas sensor
- `ZG-227ZL` TuYa Temperature & humidity LCD sensor
- `ZNCWWSQ01LM` Xiaomi Aqara pet feeder C1

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.28.3'>1.28.3</a>.</em>

**Comment by c0mplex1** [comment]
After upgrading Zigbee2MQTT to 1.28.3-1 my "[MOES 2 Gang Zigbee+RF Switch Module](https://nl.aliexpress.com/item/1005001830736248.html?gatewayAdapt=glo2nld)" has turned into a "TuYa TS011F_plug_3" and has become unusable.
Can this be fixed, or should I wait for the next update?

**Reply by Koenkk** [reply]
Fixed in 1.28.4 (just released)

**Reply by c0mplex1** [reply]
Issue is fixed.
Thank you.

**Comment by leffenke** [comment]
Hi!

I am using Openhabian OS, and apt-get doesnt offer the new releases to update.
Shall I wait for a few days and it will be available, or there is another way to upgrade?

Thanks a lot.

**Comment by walter-ve** [comment]
I see BMCT-SLZ Bosch Bosch Light/shutter control unit II (@slugzero @smarts4home) is now supported.
Running ZM2 in Home Assistant, version 1.30.4; I just connected this device,  but it still shows as 'unsupported' and it is not recognized.  Anything that I could check to make it work?
